plutus-benchmark/validation/Common.hs

```diff
@@ -131,9 +130,10 @@ benchWith act = do
 
 throughCheckScope :: UPLC.HasIndex name => UPLC.Term name uni fun a -> UPLC.Term name uni fun a
 throughCheckScope t =
-     case runExcept $ checkScope @PLC.FreeVariableError t of
+     case checkScope @PLC.FreeVariableError t of
         Right _ -> t
         Left _  -> error "scopecheck failed"
+{-# INLINE throughCheckScope #-}
```

Comment:
unnecessary: this function isn't called from the benchmarked part! And I think we could maybe just bin it.

Reply:
will be done by the other pr